src/Swarm/Version.hs

```diff
@@ -58,32 +49,12 @@ import Text.ParserCombinators.ReadP (readP_to_S)
 -- >>> import Data.Version (Version (..), parseVersion)
 -- >>> import Text.ParserCombinators.ReadP (readP_to_S)
 
-gitInfo :: Either String GitInfo
-gitInfo = $$tGitInfoCwdTry
-
-commitInfo :: String
-commitInfo = case gitInfo of
-  Left _ -> ""
-  Right git -> " (" <> giBranch git <> "@" <> take 10 (giHash git) <> ")"
-
-type CommitHash = String
-
 -- | Check that the tag follows the PVP versioning policy.
 --
 -- Note that this filters out VS Code plugin releases.
 isSwarmReleaseTag :: String -> Bool
 isSwarmReleaseTag = all (\c -> isDigit c || c == '.')
 
-tagVersion :: Maybe (CommitHash, String)
-tagVersion = case gitInfo of
```

Comment:
What happened to `tagVersion`?  Was it dead code?

Reply:
Yes, we could not use it for release. Instead, we have to make sure that the version and tag are the same.